classes/ProductJson.php

```diff
@@ -111,7 +111,7 @@ private function getSingleProduct($id_product, $id_lang)
 
     private function getProductDefaultCategory($idCategory, $idLanguage)
     {
-        if (!isset($this->categoryCache[$idCategory])) {
+        if ($idCategory && !isset($this->categoryCache[$idCategory])) {
             $this->categoryCache[$idCategory] = $this->translateCategory($idCategory, $idLanguage);
         }
         return empty($this->categoryCache[$idCategory]) ? "" : $this->categoryCache[$idCategory];
```

Comment:
I'm not sure but for safety I would use `return (!$idCategory || empty($this->categoryCache[$idCategory])) ? "" : $this->categoryCache[$idCategory];`